src/servers/src/http.rs

```diff
@@ -708,10 +709,24 @@ impl HttpServer {
                         authorize::check_http_auth,
                     )),
             )
+            .nest(
+                &format!("/{HTTP_API_VERSION}/admin"),
+                Router::new()
+                    // handler for changing log level dynamically
+                    .route(
+                        "/log_level",
+                        routing::get(dyn_log::dyn_log_handler).post(dyn_log::dyn_log_handler),
+                    ),
+            )
             // Handlers for debug, we don't expect a timeout.
             .nest(
                 &format!("/{HTTP_API_VERSION}/prof"),
```

Comment:
```suggestion
                &format!("/prof"),
```

We can agree that `/{HTTP_API_VERSION}` is for user protocol interfaces, but `/prof` and `/debug` are not. So let's remove the version prefix. cc @evenyag